parts/k8s/containeraddons/1.16/kubernetesmasteraddons-omsagent-daemonset.yaml

```diff
@@ -32,7 +32,7 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "events", "nodes", "namespaces", "services"]
   verbs: ["list", "get", "watch"]
-- apiGroups: ["extensions"]
+- apiGroups: ["apps", "extensions"]
```

Comment:
Rashmi,  I hope, you have tested already. Since there is feedback related to reconcile so can you please create a cluster with addon and test it E2E?